.travis.yml

```diff
@@ -4,15 +4,18 @@ python:
   - "3.6"
 
 install:
-  - pip install -r requirements.txt
-  - pip install codecov pytest pytest-cov
+  - pip install --upgrade pip
+  - pip install "pytest>=3.6"
+  - pip install codecov pytest-cov pytest-asyncio pylint
```

Comment:
Could we have these dependencies in `setup.py` as well?  I can't remember or find docs about why we put them here instead.

Reply:
IMO [the style](https://github.com/ethereum/py-evm/blob/master/setup.py#L31-L41) might be pretty suitable, or even put them in a separate `requirements_dev.txt`

Reply:
`Requirements_dev` is a good option, thumbs up!) I will have a chance to add it only at the late evening, though. Can we merge this code as is and add missing deps in a separate PR or something?